sdk/eventhub/Azure.Messaging.EventHubs/tests/Consumer/EventHubConsumerClientLiveTests.cs

```diff
@@ -254,10 +254,10 @@ public async Task ConsumerCanReadBatchOfEvents()
             await using (EventHubScope scope = await EventHubScope.CreateAsync(1))
             {
                 using var cancellationSource = new CancellationTokenSource();
-                cancellationSource.CancelAfter(TimeSpan.FromMinutes(4));
+                cancellationSource.CancelAfter(TimeSpan.FromMinutes(5));
```

Comment:
My goal is to be able to tune them per-test as we see different latency around service operations in the nightly runs.   For example, this one test on macOS that was consistently hitting a timeout while passing reliably on other platforms.   I value the freedom to adjust this test alone and not have to consider the ramifications across the rest.

Reply:
Makes sense. I was looking at it from the perspective of - If the response to a timeout failure is usually to increase the timeout, perhaps we should pick the max timeout we are ok with and apply that to all these live tests. In that case the response to a timeout would be to tweak the test rather than the timeout.

Reply:
The problem is that we're not necessarily in control of the timing enough that tweaking the test works.   If we wanted to send 5000 events for a test, for example, I'd prefer not to be locked into reducing that number in an effort to right the common timeout value.   Nor do I want to potentially raise the acceptable time for all tests to account for that outlier.

I'm looking at it as trying to avoid having the run extend itself in the face of service issues.   Shouldn't be a huge concern, but I also don't see a ton of benefit to having a common value.   Maybe I'm overlooking a benefit?

Reply:
The only benefit I see is avoiding the need to manually update and tweak each one in favor of drawing a line in the sand for the max duration we're willing to allow a healthy live test to execute. If we have a use case for significantly different duration profiles as in the 5000 event example, that certainly would be a valid reason to keep them individually hand tuned.

Reply:
I'm going to punt this for future consideration.   We'll likely have another tweak or two go through PR in response to observations about the nightly run as we stabilize.    Right now, we're seeing macOS be the outlier with some slowness, which may very well warrant applying the philosophy of "pick the highest possible threshold to consider success" and applying that.